batch/manage-pool/manage-pool.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Authenticate Batch account CLI session.
+az batch account login -g myresource group -n mybatchaccount
+
+# Create a new Windows PaaS pool with 3 Standard A1 VMs.
```

Comment:
We usually use the term 'cloud service platform' or 'cloud service configuration' rather than 'PaaS.'